sklearn/utils/setup.py

```diff
@@ -55,6 +55,7 @@ def configuration(parent_package="", top_path=None):
         "sklearn/utils/_seq_dataset.pxd.tp",
         "sklearn/utils/_weight_vector.pyx.tp",
         "sklearn/utils/_weight_vector.pxd.tp",
+        "sklearn/linear_model/_sag_fast.pyx.tp",
```

Comment:
This source is already generated in `linear_model`:

https://github.com/scikit-learn/scikit-learn/blob/82df48934eba1df9a1ed3be98aaace8eada59e6e/sklearn/linear_model/setup.py#L30-L32

Do you need to generate it here as well?

Reply:
i don't think so